Gifski/Tooltip.swift

```diff
@@ -40,6 +40,11 @@ final class Tooltip: NSPopover {
 		of positioningView: NSView,
 		preferredEdge: NSRectEdge
 	) {
+		guard positioningView.window != nil else {
+			assertionFailure("Tooltip must be shown from a view with a window")
+			return
+		}
```

Comment:
This is just a preventive measure to ensure the tooltip can never accidentally crash the app. Without this, `super.show` could cause AppKit to emit a fatal error.